sources/packages/backend/apps/api/src/auth/userToken.interface.ts

```diff
@@ -85,6 +85,7 @@ export interface IUserToken {
    */
   identityProvider:
     | IdentityProviders.BCSC
+    | IdentityProviders.BCeIDBasic
```

Comment:
`BCeIDBasic` does not belong in here.
Please use the `identitySpecificProvider` instead.

Reply:
From the token perspective, the current mapping is correct and should not have the `BCeIDBasic`.
We should change the DB entity model type.